config/routes.rb

```diff
@@ -6,9 +6,15 @@
   resources :attachments, only: :destroy
 
   resources :questions do
+    member do
```

Comment:
Btw, it's a great place to use [routing concerns](http://api.rubyonrails.org/classes/ActionDispatch/Routing/Mapper/Concerns.html).